src/file-manager/fm-icon-view.h

```diff
@@ -88,6 +88,7 @@ struct FMIconViewClass
 
     /* Override "clean_up" if your subclass has its own notion of where icons should be positioned */
     void	 (* clean_up)			 (FMIconView *icon_view);
+	void    (* clean_up_by_time)            (FMIconView *icon_view);
```

Comment:
Indentation here seems off - maybe you're using Tabs where the rest of the code uses Spaces?